src/domUpdates.js

```diff
@@ -31,13 +32,33 @@ export function showUserStepsVsAvg(userSteps, avg) {
 }
 
 export function showCurrentDayWaterIntake(userHydrationData, date) {
-  waterIntake.innerText = `${getDailyWater(userHydrationData, date)}`
+  showWaterGlasses(getDailyWater(userHydrationData, date))
```

Comment:
Since both calls to getDailyWater return the same value, should we store that in a variable and pass that variable in instead? To DRY things up a little bit?